### DISCUSSION TITLE
calibration load

### DISCUSSION THREAD

**Discussion by Michaelli333** [jankae/LibreVNA#222] [Q&A] [answered]
Hi Jan
when i use the SCPI command:vna.query(":VNA:CALibration:LOAD? 50M4991.CAL")  in pyhton, i get an error : The calibration kit file associated with theselected calibration could not be parsed. Thecalibration might not be accurate. (Unable toopen file). How can i handle it?
Best regards
 Li

**Comment by jankae** [comment, marked as answer]
Hi Li,
this should work, but here are some questions that hopefully help to figure out the problem:

- Which GUI and firmware version are you using?
- Can you open the same calibration file by using the menu in the GUI?
- Do you have a "50M4991.calkit" file in the same folder?
